ui/mirage/handlers/clients.js

```diff
@@ -814,11 +814,13 @@ export default function (server) {
   });
 
   server.get('sys/license/status', function () {
+    const startTime = new Date();
+
     return {
       data: {
         autoloading_used: true,
         autoloaded: {
-          expiration_time: '2022-05-17T23:59:59.999Z',
+          expiration_time: formatRFC3339(addDays(startTime, 365))
```

Comment:
Each of these lines need commas at the end. I'm curious if tests passed because this endpoint failed to build and so it didn't intercept the tests' network request...just spitballin'